ActivityPub.Types/AS/Collection/ASCollection.cs

```diff
@@ -98,6 +99,32 @@ public int TotalItems
     public bool HasItems => Items?.Any() == true;
 
     public static implicit operator ASCollection(List<ASObject> collection) => new() { Items = new LinkableList<ASObject>(collection) };
+
+    public IEnumerator<Linkable<ASObject>> GetEnumerator()
+    {
+        if (Items == null)
+        {
+            yield break;
+        }
+
+        foreach (var item in Items)
+        {
+            if (item.HasValue)
+            {
+                yield return item.Value;
+            }
+
+            if (item.HasLink)
+            {
+                yield return item.Link;
+            }
+        }
+    }
+
+    IEnumerator IEnumerable.GetEnumerator()
```

Comment:
Would you mind switching this to expression syntax?
```suggestion
    IEnumerator IEnumerable.GetEnumerator() => GetEnumerator();
```